library/std/src/net/ip.rs

```diff
@@ -1540,8 +1545,8 @@ impl Ipv6Addr {
     /// assert_eq!(Ipv6Addr::new(0xff00, 0, 0, 0, 0, 0, 0, 0).octets(),
     ///            [255, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
     /// ```
+    #[rustc_const_stable(feature = "const_ipv6", since = "1.50.0")]
     #[stable(feature = "ipv6_to_octets", since = "1.12.0")]
-    #[rustc_const_stable(feature = "const_ipv6", since = "1.32.0")]
```

Comment:
This one was already stable since 1.32. Did you change it to 1.50 on purpose?

Reply:
Oops good catch. I only intended to change the ordering of the attributes to make it consistent with all the other ones.